react/features/large-video/components/LargeVideo.web.js

```diff
@@ -361,7 +383,9 @@ function _mapStateToProps(state) {
         _verticalFilmstripWidth: verticalFilmstripWidth.current,
         _verticalViewMaxWidth: getVerticalViewMaxWidth(state),
         _visibleFilmstrip: visible,
-        _whiteboardEnabled: isWhiteboardEnabled(state)
+        _whiteboardEnabled: isWhiteboardEnabled(state),
```

Comment:
Can you order (move up) the prop alphabetically, also in the `Type Props`

I got this also from other reviews on my PRs so maybe it's good to keep this rule